src/Engine.cpp

```diff
@@ -1032,7 +1032,7 @@ bool Engine::update_entity_status(
                     {
                         backend::StatusLevel entity_status = backend_connection_.get_status(id);
                         auto entity_item = entity_status_model_->getTopLevelItem(
-                            id, backend_connection_.get_name(id), entity_status, description, entity_guid);
+                            id, entity_kind + "_" + backend_connection_.get_name(id), entity_status, description, entity_guid);
```

Comment:
I'd prefer something more human-readable like this

```suggestion
                            id, entity_kind + ": " + backend_connection_.get_name(id), entity_status, description, entity_guid);
```